### DISCUSSION TITLE
Intermediate CSS Concepts - Exercises

### DISCUSSION THREAD

**Discussion by ghost** [TheOdinProject/curriculum#26418] [General]
Hi,

I'm currently working on the [Intermediate CSS Concepts](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#intermediate-css-concepts).

To be honest, it's all about reading without any practical exercises. Yes, I can practice on my own, but there's already a CSS exercises repo at https://github.com/TheOdinProject/css-exercises
Is there a possibility to add additional exercises covering these lessons?

**Comment by ManonLef** [comment]
I do think it's expected, as you are in foundations, to practice along with the reading material. By having VSCode open and implementing or testing the ideas presented, they will stick much better. 

What kind of exercises did you personally have in mind?

**Reply by ghost** [reply]
Absolutely, most of us, including myself, find the best way to learn is by practicing on our own. However, considering the existing exercises repository, expanding it seems like a great solution. It would fill the gaps, provide a handy reference, and likely reduce the number of questions on Discord etc
Alternatively, we could delete all existing exercises, simplifying it all by having a clear message at the bottom of the page, encouraging self-practice.

Similar to one already existing https://github.com/TheOdinProject/css-exercises

I created one for example aswell
https://github.com/sovzl/wdym
